src/Sign.Core/CertificateServices/CertificateStoreService.cs

```diff
@@ -134,8 +137,8 @@ private Task<X509Certificate2> GetStoreCertificateAsync()
             // Search User or Machine certificate stores.
```

Comment:
I can't comment directly on (new) line 134 because it wasn't changed; however, [you updated the `CertificateNotFoundInFile` resource string to take 2 string replacements](https://github.com/dotnet/sign/pull/691/files#diff-f0243fdd6fbbfb540457054900a66fd592aca55b3410d47e651313b479207b43R135), but you're still passing one argument here.